test/integration/highlight_editor_spec.mjs

```diff
@@ -2144,4 +2145,202 @@ describe("Highlight Editor", () => {
       );
     });
   });
+
+  fdescribe("Annotation pop-up has the expected behaviour", () => {
```

Comment:
This needs to be changed to describe, keeping it fdescribe for now, so it's easier to review!